### PR TITLE
featu(search) Use snuba to fetch events by default

### PR DESCRIPTION
**Pull request by fpacifici**
snuba.events-queries.enabled option is supposed to be already active in production. This commit activates it by default and fixes the tests that did not support it.
This step is needed in order to remove our dependency on PG events which is useful for issueless
events.

### CODE REVIEW

**Review by mattrobenolt — COMMENTED** [review]
Why not just remove the switch instead? If it’s defaulted to true, there’s 0 reason for someone to turn it off.

**Review by mitsuhiko — APPROVED** [review]
lgtm but i would indeed remove all the now dead code that hangs on this switch.

**Comment by fpacifici** [comment]
I cannot remove the switch yet, because there is a handful of tests that force it to False, that I still have to fix. I will address them in a separate PR.

File: tests/sentry/api/endpoints/test_event_attachment_details.py [hunk 1]
```diff
@@ -13,12 +14,15 @@ class CreateAttachmentMixin(object):
...
-        self.group = self.create_group(project=self.project, first_release=self.release)
-        self.event = self.create_event(
-            event_id='a',
-            group=self.group,
-            datetime=datetime(2016, 8, 13, 3, 8, 25),
-            tags={'sentry:release': self.release.version}
+        min_ago = (timezone.now() - timedelta(minutes=1)).isoformat()[:19]
+        self.event = self.store_event(
+            data={
+                # 'event_id': 'a' * 32,
```
Comment (markstory) [inline tests/sentry/api/endpoints/test_event_attachment_details.py]: You could remove this :bike:

File: tests/sentry/api/endpoints/test_event_attachments.py [hunk 2]
```diff
@@ -12,23 +13,24 @@ class EventAttachmentsTest(APITestCase):
...
-            group=group,
-            datetime=datetime(2016, 8, 13, 3, 8, 25),
-            tags={'sentry:release': release.version}
+        min_ago = (timezone.now() - timedelta(minutes=1)).isoformat()[:19]
+        event1 = self.store_event(
+            data={
+                # 'event_id': 'a' * 32,
+                'fingerprint': ['group1'],
+                'timestamp': min_ago,
+                # 'tags': {'sentry:release': release.version},
```
Comment (markstory) [inline tests/sentry/api/endpoints/test_event_attachments.py]: More commented code that could be removed.

**Review by markstory — APPROVED** [review]

File: tests/sentry/api/endpoints/test_event_attachments.py (hunk 2 above)
Comment (fpacifici) [reply tests/sentry/api/endpoints/test_event_attachments.py]: weird, I thought I removed all of them. Will fix

**Pull request merged**